plugins/inputs/prometheus/README.md

```diff
@@ -10,50 +10,56 @@ in Prometheus format.
 [[inputs.prometheus]]
   ## An array of urls to scrape metrics from.
   urls = ["http://localhost:9100/metrics"]
-  
+
   ## Metric version controls the mapping from Prometheus metrics into
   ## Telegraf metrics.  When using the prometheus_client output, use the same
   ## value in both plugins to ensure metrics are round-tripped without
   ## modification.
   ##
-  ##   example: metric_version = 1; 
+  ##   example: metric_version = 1;
   ##            metric_version = 2; recommended version
   # metric_version = 1
-  
+
   ## Url tag name (tag containing scrapped url. optional, default is "url")
   # url_tag = "url"
-  
+
   ## Whether the timestamp of the scraped metrics will be ignored.
   ## If set to true, the gather time will be used.
   # ignore_timestamp = false
-  
+
   ## An array of Kubernetes services to scrape metrics from.
   # kubernetes_services = ["http://my-service-dns.my-namespace:9100/metrics"]
-  
+
   ## Kubernetes config file to create client from.
   # kube_config = "/path/to/kubernetes.config"
-  
+
   ## Scrape Kubernetes pods for the following prometheus annotations:
   ## - prometheus.io/scrape: Enable scraping for this pod
   ## - prometheus.io/scheme: If the metrics endpoint is secured then you will need to
   ##     set this to 'https' & most likely set the tls config.
   ## - prometheus.io/path: If the metrics path is not /metrics, define it with this annotation.
   ## - prometheus.io/port: If port is not 9102 use this annotation
   # monitor_kubernetes_pods = true
-  
+
   ## Get the list of pods to scrape with either the scope of
   ## - cluster: the kubernetes watch api (default, no need to specify)
   ## - node: the local cadvisor api; for scalability. Note that the config node_ip or the environment variable NODE_IP must be set to the host IP.
   # pod_scrape_scope = "cluster"
-  
+
   ## Only for node scrape scope: node IP of the node that telegraf is running on.
   ## Either this config or the environment variable NODE_IP must be set.
   # node_ip = "10.180.1.1"
- 
+
   ## Only for node scrape scope: interval in seconds for how often to get updated pod list for scraping.
   ## Default is 60 seconds.
   # pod_scrape_interval = 60
-  
+
+  ## Pod label & annotation tag template
+  ## Default is "{{ . }}"
+  ## Useful to glob match labels & annotations in "tagexclude" or "taginclude"
+  # pod_label_template = "pod.label/{{ . }}"
+  # pod_annotation_template = "pod.annotation/{{ . }}"
```

Comment:
Would it be simpler to make these prefixes instead of templates? That way users wouldn't need to worry about template syntax when they're prepending static strings.